lab/05-deploying-to-azure.md

```diff
@@ -6,80 +6,58 @@ We've tested our project locally and now we are ready to deploy it to Azure.
 
 ## Sign up for Azure
 
-If you haven't yet, you will need to create an Azure Account. If you don't have an account already you can sign up using <https://aka.ms/azurefree> to get up to $200 in credits. Also we've configured this application to use the minimal resources.
+If you haven't yet, you will need to create an Azure Account. If you don't have an account already you can sign up using <https://aka.ms/azurefree> to get up to $200 in credits. We've configured this application to use the minimal resources, but not all the resources used have a free tier. You can use the credits to pay for the resources used in this lab. 
```

Comment:
I'm trying to make it explicit that it's not 100% free. Might scare some folks off but better than them being mad they accrued costs, I reckon?